3rdParty/CMakeLists.txt

```diff
@@ -322,6 +322,14 @@ add_subdirectory(json-schema-validation)
 
 find_package(Threads REQUIRED)
 find_package(ZLIB REQUIRED)
+# Variables for LAPACK and BLAS
+if (NOT STATIC_EXECUTABLES)
+  set(BLA_STATIC Off)
+  set(BLA_VENDOR OpenBLAS)
+  find_package(OpenMP)
+  find_package(LAPACK)
+  find_package(BLAS)
+endif()
```

Comment:
I guess this won't disable (dynamic) linking to `clien-tools` target. Do we really need all these dependencies to arangosh, arangodump, etc.?

Reply:
No, it won't. I am still not sure how arangosh get linked with these, since they are only used in arangostorage library. I will look into it

Reply:
Hmm, so this is unfortunately like this for all tools. I have built the dependency graph and it can be seen that faiss->arango_rocksdb->arango_storage_engine->arango_replication2->arango->arangosh.
Here is arangosh.dot file for graphviz that shows that dependency. I will solve the tsan issues with this PR, and will look into why does it link these dependencies later
[arangosh.zip](https://github.com/user-attachments/files/18276073/arangosh.zip)